packages/x-charts/src/LineChart/LinePlot.tsx

```diff
@@ -21,13 +20,15 @@ export function LinePlot() {
     return null;
   }
   const { series, seriesOrder, stackingGroups } = seriesData;
-  const { xAxis, yAxis } = axisData;
+  const { xAxis, yAxis, xAxisIds, yAxisIds } = axisData;
+  const defaultXAxisId = xAxisIds[0];
+  const defaultYAxisId = yAxisIds[0];
 
   const seriesPerAxis: { [key: string]: LineSeriesType[] } = {};
 
   seriesOrder.forEach((seriesId) => {
-    const xAxisKey = series[seriesId].xAxisKey; // ?? DEFAULT_X_AXIS_KEY;
-    const yAxisKey = series[seriesId].yAxisKey; // ?? DEFAULT_Y_AXIS_KEY;
+    const xAxisKey = series[seriesId].xAxisKey; // ?? defaultXAxisId;
+    const yAxisKey = series[seriesId].yAxisKey; // ?? defaultYAxisId;
```

Comment:
Is it still relevant? 🤔 
```suggestion
    const xAxisKey = series[seriesId].xAxisKey;
    const yAxisKey = series[seriesId].yAxisKey;
```

Reply:
No, I will cut it  ✂️ 👍